pkg/cmd/server/api/helpers.go

```diff
@@ -241,6 +241,14 @@ func GetMasterFileReferences(config *MasterConfig) []*string {
 
 		refs = append(refs, &config.KubernetesMasterConfig.ProxyClientInfo.CertFile)
 		refs = append(refs, &config.KubernetesMasterConfig.ProxyClientInfo.KeyFile)
+
+		for _, key := range []string{"cluster-signing-cert-file", "cluster-signing-key-file"} {
```

Comment:
Have we done this for any other flags?  It rubs me the wrong way to make some of these special.  I think we'd need some way of staying in sync with a test at the very least.